db/migrate/20180603194914_rename_armenian_locale_key.rb

```diff
@@ -0,0 +1,9 @@
+class RenameArmenianLocaleKey < ActiveRecord::Migration[5.1]
+  def up
+    User.where(language: 'hy').update_all(language: 'hye')
+  end
+
+  def down
+    User.where(language: 'hye').update_all(language: 'hy')
```

Comment:
- Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.
- Rails/SkipsModelValidations: Avoid using `update_all` because it skips validations.